README.md

```diff
@@ -72,21 +72,20 @@ fn main() {
                     let size = window.inner_size();
                     (size.width, size.height)
                 };
-                let buffer = (0..((width * height) as usize))
-                    .map(|index| {
-                        let y = index / (width as usize);
-                        let x = index % (width as usize);
-                        let red = x % 255;
-                        let green = y % 255;
-                        let blue = (x * y) % 255;
+                surface.resize(width, height).unwrap();
```

Comment:
Do we want to resize on each render? Resizing on each resize, or perhaps checking for different sizing, is probably more reasonable. At present the cost of an extraneous resize is minimal, but that isn't documented as being the case so we should not rely on it. Documenting that the cost of a resize will be minimal is also an option, but this constricts future use so I advocate just fixing the examples (there are other examples that have this problem).